### DISCUSSION TITLE
Unbuild: Dynamic imports

### DISCUSSION THREAD

**Discussion by davidabou** [unjs/community#6] [Q&A] [answered]
Hi, does anyone know how to dynamically import TypeScript files with `unbuild` in ESM? I have this code:

```ts
export const command = {
  ping: () => import(findPath('ping')).then(m => m.default || m)
}
```

However, during the prepack with `unbuild`, I don't see a 'chunks' folder, and consequently there's an import issue for this file.

**Comment by davidabou** [comment]
When the argument of import is a static string like './commands/ping' the chunks folder is generated on prepack and everything works well but with a variable, in my case `findPath('ping')` the chuncks folder is not generated

**Comment by pi0** [comment, marked as answer]
https://github.com/unjs/unbuild/issues/395
